certified-connectors/Vonage/README.md

```diff
@@ -0,0 +1,16 @@
+## Vonage Connector
```

Comment:
Readme doenst have all required sections. Please refer to this [link ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for a readme template.

Reply:
Looking at these now

Reply:
I have updated this with all required sections